src/encoded/static/scss/encoded/modules/_matrix.scss

```diff
@@ -15,6 +18,390 @@ $row-data-header-width: 200px;
+    }
+    .test-project-selector {
+        label {
+            margin-left: 5px;
+        }
+    }
+    .matrix-header {
+        border-bottom: 0;
+    }
+    .header-clear-links {
+        margin-top: 20px;
+        @media screen and (max-width: $screen-sm-min) {
+            margin: 10px 5px;
+        }
+        button {
+            position: relative;
+            top: 0;
+            bottom: 0;
+            right: 0;
+            left: 0;
+            @media screen and (max-width: $screen-sm-min) {
+                width: auto;
+            }
+        }
+    }
+    .tab-nav-Homosapiens {
+        border-bottom: 2px solid $Homo-sapiens;
+    }
+    .tab-nav-Musmusculus {
+        border-bottom: 2px solid $Mus-musculus;
+    }
+    .nav-tabs {
+        li {
+            margin-bottom: -1px;
+            &.active {
+                border-bottom: 0;
+                .organism-button {
+                    font-weight: 800;
+                }
+            }
+            .organism-button {
+                background-color: unset;
+                border: 0;
+                padding: 3px 10px;
+                margin: 0;
+                font-weight: 600;
+                &:hover {
+                    background-color: unset;
+                }
+            }
+            a[disabled] {
+                pointer-events: none;
+                cursor: unset;
+                opacity: 0.7;
+                .organism-button {
+                    color: #828282;
+                }
+                img {
+                    opacity: 0.5;
+                }
+            }
+        }
+        .Homosapiens-tab {
+            &.active {
+                a {
+                    border-right: 3px solid $Homo-sapiens;
+                    border-left: 3px solid $Homo-sapiens;
+                    border-top: 3px solid $Homo-sapiens;
+                }
+            }
+            .organism-button {
+                color: $Homo-sapiens;
+            }
+        }
+        .Musmusculus-tab {
+            &.active {
+                a {
+                    border-right: 3px solid $Mus-musculus;
+                    border-left: 3px solid $Mus-musculus;
+                    border-top: 3px solid $Mus-musculus;
+                }
+            }
+            .organism-button {
+                color: $Mus-musculus;
+            }
+        }
+    }
+    .matrix-header__filter-controls {
+        display: block;
+        flex: unset;
+        margin: 10px 0 0;
+    }
+    .matrix-general-search {
+        display: flex;
+        align-items: baseline;
+        flex-wrap: wrap;
+        text-align: left;
+        input[type="search"] {
+            font-size: 0.8rem;
+        }
+        @media screen and (max-width: $screen-sm-min) {
+            font-size: 0.9rem;
+            margin-bottom: 0;
+        }
+        @media screen and (max-width: $screen-xs-min) {
+            font-size: 0.8rem;
+            margin-bottom: 20px;
+        }
+        i {
```

Comment:
This is for my personal curiosity. Have you ever seen anyone remove the italics via CSS from i-html tag?

Reply:
No, I'm not sure to what you're referring?

Reply:
I missed this comment. i HTML used to be used for italic. That was before CSS took over that role. People latter highjacked it and started using it for icons. I was just curious if you see people remove the italic-functionality in it.